develop/tutorials/articles/280-workflow/01-creating-workflow-definitions/03-notifications.markdown

```diff
@@ -0,0 +1,78 @@
+# Workflow Notifications
+
+While an asset is in a workflow, it's important that relevant Users are notified
+about certain events, like when a review task is completed. Any workflow node
+that can take an `<actions>` element can have notifications.
+
+    <actions>
+        <action>
+            <notification>
+                <name>Creator Modification Notification</name>
+                <template>Your submission was rejected by ${userName}, please modify and resubmit.</template>
+                <template-language>freemarker</template-language>
+                <notification-type>email</notification-type>
+                <notification-type>user-notification</notification-type>
+                <execution-type>onAssignment</execution-type>
+            </notification>
+        </actions>
+    </actions>
+
+## Notification Options
+
+There are several elements that can be specified in a `<notification>`:
+
+**Name**
+: Set the name of the notification in the `<name>` element. This information is
+used to display the notification in the _My Workflow Tasks_ widget of a User's
+personal site.
+
+**Template**
+: The `<template>` element contains the actual message of the notification. The
+syntax is determined by the template language you're using. 
+
+**Template Language**
+: Choose from `freemarker`, `velocity`, or plain `text` in the
+`<template-language>` tag. 
+
+**Notification Type**
+: Choose whether to send an `email`, `user-notification` (via the Notification
+widget), `im` (instant message), or `private-message` in the
+`<notification-type>` tag.
+
+        <notification-type>email</notification-type>
+
+**Execution Type**
+: Choose to link the sending of the notification to entry into the node
+(`onEntry`), when a task is assigned (`onAssignment`), or when the workflow
+processing is leaving a node (`onExit`). 
+
+**Recipients**
```

Comment:
Besides the already mentioned _Recipients_, I think that worth tell the reader all the available options, like _Role_, _Role Type_, etc.